pkgs/misc/cups/libcupsfilters.nix

```diff
@@ -0,0 +1,102 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, avahi
+, cups
+, dejavu_fonts
+, dbus
+, freetype
+, fontconfig
+, ghostscript
+, lcms2
+, libexif
+, libjpeg
+, libpng
+, libtiff
+, pkg-config
+, poppler_utils
+, mupdf # mutool
+, poppler
+, qpdf
+, withAvahi ? true
```

Comment:
What would be the usecase for this option? If there is not really one we might as well remove it.

Reply:
I took the `withAvahi` logic from `cups-filters`, where it was added in 9a93279738708df0801185a35fb31671b4616b70 (PR  #148257), so I'm assuming there's a purpose to it